scripts/index.js

```diff
@@ -1,4 +1,27 @@
-const cards = document.querySelectorAll('.card');
+import { CardAdptive } from './CardAdaptive.js';
+import { CardDesctop } from './CardDesctop.js';
+import { initialCardItems } from './cardItems.js';
+
+const cardList = document.querySelector('.project__list');
+
+const template = {
+  card: '#card-item-template',
+  toggleAdaptive: '#card-toggle-adaptive-template',
+  toggleDesctop: '#card-toggle-desctop-template'
+};
+
+if(cardList) {
+  initialCardItems.forEach((item) => {
+    if(item.linkImgMobile) {
+      const cardAdptive = new CardAdptive(item, template);
```

Comment:
Опечатки сильно напрягают)

```suggestion
      const cardAdaptive = new CardAdaptive(item, template);
```

Reply:
Исправил